smart_campus/app/templates/app/travelplan_edit_page.html

```diff
@@ -0,0 +1,129 @@
+{% extends 'app/base.html' %}
+{% load static %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.css">
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.js"></script>
```

Comment:
Move it to script block

Reply:
Okay...
I am curious why this didn't merge in.